tests/test-definition.yaml

```diff
@@ -10,6 +10,10 @@ dimensions:
   - name: keycloak
     values:
       - 23.0.1
+  - name: openshift
+    values:
+      - "true"
+      - "false"
```

Comment:
```suggestion
  - name: k8s-flavour
    values:
      - vanilla
      - openshift
```

Reply:
We have this pattern of openshift true/false in other tests and I would like to stick with it for consistency.

Reply:
I like @nightkr's suggestion in general, but have seen the other pattern throughout. Maybe it can change separately?

Reply:
Alright.

Reply:
I agree that this way of doing it would be prettier too. I'll resolve this then.